### ISSUE TITLE
No pillager outpost where Amidst says there is

### ISSUE THREAD

**Issue by merukit** [toolbox4minecraft/amidst#928]
Amidst 4.6
Seed: 9058136630944956755
X, Z Coords: [1284, 372]
World generated in 1.16.4 and Amidst set to 1.16.4 (latest version)

It's just a taiga biome with no outpost there even though the program does think there is one.

I saw this brought up before but those issues were closed.

**Comment by scarifyFX** [comment]
I generated the world using the seed and when I did /seed I got 5418114931809310958. I will try generating the world using a server to get an accurate seed world generation.
![Skärmbild (62)](https://user-images.githubusercontent.com/47931007/103909790-6455dd80-5104-11eb-9ddb-54166a433d55.png)
![2021-01-07_16 21 06](https://user-images.githubusercontent.com/47931007/103909823-6e77dc00-5104-11eb-8fba-0d22de53602b.png)
I teleported to a village nearby which I found on amidst and nothing was wrong with it, using chunk base I did not see an outpost.
